hw/top_earlgrey/dv/verilator_sim_cfg.hjson

```diff
@@ -56,13 +56,14 @@
   run_modes: [
     {
       name: sw_test_mode
-      sw_images: ["sw/device/boot_rom/boot_rom:0"]
+      sw_images: ["sw/device/boot_rom/boot_rom:0",
```

Comment:
Is this working? Thanks for factoring this in too :)

Reply:
I made the changes, but this is the only flow that I have not actively verified by inspecting the waves (I checked the regular verilator invocation via fusesoc, so the `--meminit` switches do work as intended).
I believe we are not calling this CFG file in our regressions, right?

Reply:
You can give this a try right away:
```console
$ ./util/dvsim/dvsim.py hw/top_earlgrey/dv/verilator_sim_cfg.hjson -i all -mp 1
```

Reply:
Looks like this has an issue (same error on master, so it seems to be independent of this change).
Need to take a closer look.

## List of Failures

**BUILD:** default<br>
**LOG:** $scratch_path/default/build.log<br>
Last 10 lines of the log:<br>
```
[Executing]:
make -f /home/msf/Projects/opentitan/hw/dv/tools/dvsim/sim.mk build build_cmd="fusesoc --cores-root /home/msf/Projects/opentitan run" build_dir="/home/msf/Projects/opentitan/scratch/otp-preload-integration/top_earlgrey_verilator-sim-verilator/default" build_opts="--flag=fileset_top --target=sim --build-root=/home/msf/Projects/opentitan/scratch/otp-preload-integration/top_earlgrey_verilator-sim-verilator/default --setup --build lowrisc:systems:top_earlgrey_verilator:0.1" post_build_cmds="" pre_build_cmds="[ -d /home/msf/Projects/opentitan/hw/foundry ] && \
   touch /home/msf/Projects/opentitan/hw/foundry/FUSESOC_IGNORE" proj_root="/home/msf/Projects/opentitan" sv_flist_gen_cmd="" sv_flist_gen_dir="/home/msf/Projects/opentitan/scratch/otp-preload-integration/top_earlgrey_verilator-sim-verilator/default" sv_flist_gen_opts=""

[make]: pre_build
make: *** [/home/msf/Projects/opentitan/hw/dv/tools/dvsim/sim.mk:17: pre_build] Error 1
```

Reply:
It looks like the prebuild command does only work correctly if the `hw/foundry` directory is present.
I modified that command a bit such that it also works when that directory is absent. PTAL